iOSTest/View Controllers/LoginViewController.swift

```diff
@@ -70,33 +67,29 @@ class LoginViewController: UIViewController {
     
     func validateUsernameAndPassword() {
 
-        let validUsername = "user@iosstudies.co"
-        let validPassword = "pinksalmon"
-            
+        username = usernameTextfield.text
+        password = passwordTextfield.text
+        
         // Check validity of username and password
         if (username != validUsername) || (password != validPassword) {
             
-            let invalidAlert = UIAlertController(title: "Invalid Username or Password", message: "Please enter a valid username and passowrd.", preferredStyle: .alert)
-            let invalidAction = UIAlertAction(title: "OK", style: .default, handler: nil)
+            let alert = UIAlertController(title: "Invalid Username or Password", message: "Please enter a valid username and passowrd.", preferredStyle: .alert)
+            let action = UIAlertAction(title: "OK", style: .default, handler: nil)
```

Comment:
Since there are 2 different types of alerts, you can undo the changes and use "invalidAlert" and "invalidAction". Apologies for the mix up.